### PR TITLE
Content tweaks and Typos for Components area

### PR DESCRIPTION
**Pull request by isralduke**
This PR:

- updates content in Components
- updates some includes for Components

### CODE REVIEW

File: docs/components/loading-animation.md [hunk 1]
```diff
@@ -19,8 +19,18 @@ eleventyNavigation:
 
 ## Usage
 
-{% include 'markup/spinner.njk' %}
+<div class="spinner-border text-info" role="status">
+  <span class="sr-only">Loading...</span>
+</div>
+<div class="spinner-border text-ui" role="status">
+  <span class="sr-only">Loading...</span>
+</div>
```
Comment (frogpad7) [inline docs/components/loading-animation.md:27]: Why get rid of the include statement? If this is too small of a file for this then there are many more that could follow this same presentation.

File: docs/_includes/markup/buttons-previous-next-page.njk [hunk 2]
```diff
@@ -1,14 +1,14 @@
-<div class="btns-previous-forward">
+<div class="btns-previous-next">
```
Comment (frogpad7) [inline docs/_includes/markup/buttons-previous-next-page.njk:1]: Not sure why this does not have the css update with it to verify it appears correctly. Can you combine the updates or place this with the css branch?

**Review by frogpad7 — CHANGES_REQUESTED** [review]
Not entirely sure about a few of the markup changes being made; please verify their purpose and update this branch accordingly.

File: docs/components/loading-animation.md (hunk 1 above)
Comment (isralduke) [reply docs/components/loading-animation.md:27]: it makes editing quicker as this is only used in once place. We can make those changes as we go forward.

**Review by frogpad7 — APPROVED** [review]
The Prev & Next buttons now look spectacular!!

**Pull request merged**